model/riscv_sys_regs.sail

```diff
@@ -487,7 +487,7 @@ register mepc : xlenbits
  * When misa.C is writable, it zeroes only xepc[0].
  */
 function legalize_xepc(v : xlenbits) -> xlenbits =
-  /* allow writing xepc[1] only if misa.C is enabled or could be enabled 
+  /* allow writing xepc[1] only if misa.C is enabled or could be enabled
```

Comment:
Don't touch unrelated lines

Reply:
It would be helpful to add a pre-commit config that strips whitespace and enable it in CI. I can set that up if you'd like?